src/AccordionCollapse.js

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import Collapse from './Collapse';
+import { useBootstrapPrefix } from './ThemeProvider';
+import AccordionContext from './AccordionContext';
+
+const propTypes = {
+  /** @default 'accordion-collapse' */
+  bsPrefix: PropTypes.string,
+  /**
+   * A key that corresponds to the toggler that triggers this collapse's expand or collapse.
+   */
+  eventKey: PropTypes.string.isRequired,
+};
+
+const AccordionCollapse = React.forwardRef(
+  ({ bsPrefix, children, className, eventKey, ...props }, ref) => {
+    bsPrefix = useBootstrapPrefix(bsPrefix, 'accordion-collapse');
```

Comment:
i don't think this is actually a class in bootstrap. perhaps this only needs to handle collapsing, per https://getbootstrap.com/docs/4.3/components/collapse/#accordion-example ?

Reply:
So don't apply any new classes? or would we still want to apply `collapse` as one of the classes?

Reply:
As a rule we don't creat new classes that don't exist in bootstrap since we don't alter the case. If the upstream component adds collapse then we should too otherwise no